src/tools/other/pb2nc/pb2nc.cc

```diff
@@ -1345,17 +1345,21 @@ void process_pbfile(int i_pb) {
          }
       }
 
+      // Initialize for CAPE and PBL
       if (cal_cape || cal_mlcape) {
          cape_level = 0;
+         cape_qm = bad_data_float;
```

Comment:
@hsoh-u I added code to re-initialize cape_qm and pbl_qm to bad data on lines 1351 and 1357, respectively.
So basically after each bufr message is read by `readpb_` we reinitialize them. Can you please confirm that this is correct?

Reply:
@hsoh-u I don't fully understand your comment above. Could you perhaps just commit proposed changes to the initialization of cape_qm and pbl_qm directly to the `feature_2278_qm_docs` branch?